build/parameters.go

```diff
@@ -0,0 +1,7 @@
+package build
```

Comment:
The last backfire from importing Lotus. It should embed a parameters.json file in the executable because some `init()` function in some package checks it.  Also a PITA... but thankfully this isn't important for anything else. 

These nifty things are dealt with by the Dockerfiles already, or some other tooling as Makefile. 
I thought about forking Lotus and adjusting things to avoid this stuff, but that should be well worth since would mean maintaining our own fork with changes and updating it with some frequency. 

At the moment, chose to avoid that. Seeing the big picture of this PR, there are only 2 things that are weird: faking proof files, and embedding this thing. So, it isn't that bad to take such drastic decision.